etcdctl/ctlv3/command/printer.go

```diff
@@ -159,6 +161,14 @@ func (s *simplePrinter) Watch(resp v3.WatchResponse) {
 	}
 }
 
+func (s *simplePrinter) TimeToLive(resp v3.LeaseTimeToLiveResponse) {
+	ks := make([]string, len(resp.Keys))
+	for i := range resp.Keys {
+		ks[i] = string(resp.Keys[i])
+	}
+	fmt.Printf("lease %016x granted with TTL(%ds), remaining(%ds), attached keys(%v)\n", resp.ID, resp.GrantedTTL, resp.TTL, ks)
```

Comment:
if len(resp.keys) == 0, only print granted and remaining? it's sort of misleading to do --keys=false and then it says attached keys == []